README.md

```diff
@@ -27,7 +27,7 @@ You can use generic launcher with appropriate parameter to connect with pre-conf
 Example:
   - `./bin/mantis-launcher etc` - for joining Ethereum Classic network
 
-Possible networks: `etc`, `eth`, `mordor`, `testnet-internal`
+Possible networks: `etc`, `eth`, `mordor`, `testnet-internal`. On a fresh Linux instance, it's recommended to use 8GB of memory to fully sync ETC, and have at least 100GB of disk space. Also `default-jdk` can be used if Java is not installed by default on the system.
```

Comment:
`default-jdk` can change in time & might vary between linux distros
What `java -version` did you used?

Reply:
I’ll check it out tomorrow on my server. Do you recommend I just change this section or remove it entirely and just leave hardware requirements?

Reply:
@antsankov thanks for your contribution! 

Just 2 cents: I do not have `default-jdk` on `Ubuntu`.
But I'm not sure if JDK (not JRE) is required.

Mantis docker image is already shipped with JRE of course. 
We could probably build .deb or .rpm / .rpmi package that depends on JRE. WDYT?